force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +100,17 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
-
+        List<String> wordList = sentence.split(' ', 0);
         // Create a set to store the unique words
-
+        Set<String> wordsSet = new Set<String>();
         // Add each word to the set
-
+        for (Integer i = 0; i < wordList.size(); i++) {
+            wordsSet.add(wordList.get(i));
+        }
         // Join the unique words back into a sentence
-
+        String newSentence = String.join(new List<String>(wordsSet),' ');
         // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+        return newSentence; 
```

Comment:
You could pass a list of Strings directly into to a set constructor and save yourself a lot of code. 

```
  public static String removeDuplicateWords(String sentence) {
        Set<String> uniqueWords = new Set<String>(sentence.split(' '));
        return String.join(uniqueWords, ' ');
    }
    ```